compiler/qsc_frontend/src/resolve.rs

```diff
@@ -463,20 +510,37 @@ impl Resolver {
         }
     }
 
-    fn bind_open(&mut self, name: &ast::Ident, alias: &Option<Box<ast::Ident>>) {
-        let alias = alias.as_ref().map_or("".into(), |a| Rc::clone(&a.name));
-        if self.globals.namespaces.contains(&name.name) {
+    fn bind_open(&mut self, name: &Idents, alias: &Option<Box<Ident>>) {
+        let Some(id) = self.globals.find_namespace(name.str_iter()) else {
+            self.errors.push(Error::NotFound(
+                name.iter()
+                    .map(|x| x.name.to_string())
+                    .collect::<Vec<_>>()
+                    .join("."),
+                name.span(),
+            ));
+            return;
+        };
+        let alias = alias
+            .as_ref()
+            .map_or(name.into(), |a| vec![Rc::clone(&a.name)]);
```

Comment:
I'm noticing that the default value for aliases changed here. It use to default to an empty string, which makes sense because in that case there is no alias. Now, the default is to have the alias match the regular namespace name. Looking at how this is used, as a key to the opens entries map, I'm not actually sure how this worked before, do you know? It seems like whenever an open without an alias is used, it would _always_ push a new open?